php/class-sync.php

```diff
@@ -92,6 +92,7 @@ class Sync implements Setup, Assets {
 		'syncing'        => '_cloudinary_syncing',
 		'downloading'    => '_cloudinary_downloading',
 		'process_log'    => '_process_log',
+		'process_log_v3' => '_cloudinary_process_log',
```

Comment:
@pereirinha - perhaps change the old one to `process_log_v2` or `process_log_legacy` and the new one to just `process_log` this will prevent the version issues when we get to v4.
Although I suspect, this will have implications on where `process_log` is being used elsewhere.